data/src/main/java/co/nimblehq/compose/crypto/data/service/ApiService.kt

```diff
@@ -11,7 +11,7 @@ import retrofit2.http.Query
 interface ApiService {
     @GET("coins/markets")
     suspend fun getCoins(
-        @Query("ids") coinIds: String,
+        @Query("ids") coinIds: String? = null,
```

Comment:
we have default value in Repository implementation so we could get rid of it here.